applications/visualizer/frontend/src/components/viewers/ThreeD/ThreeDViewer.tsx

```diff
@@ -67,7 +70,8 @@ function ThreeDViewer() {
   return (
     <>
       <DatasetPicker datasets={dataSets} selectedDataset={selectedDataset} onDatasetChange={setSelectedDataset} />
-      <Canvas style={{ backgroundColor: SCENE_BACKGROUND }} frameloop={"demand"}>
+      <Canvas style={{ backgroundColor: SCENE_BACKGROUND }} gl={{ alpha: true, antialias: true, preserveDrawingBuffer: true }}>
```

Comment:
note: The introduction of the preserveDrawingBuffer and removal of the on demand rendering is not ideal performance wise. Maybe it would make sense to create a tech debt card to explore alternatives

Reply:
Thanks for the tests @Salam-Dalloul  and thanks for spotting this @afonsobspinto :pray: